benchmarks/DeepLearning/Layers/SelfAttention/GoogleBenchmarkMain.cpp

```diff
@@ -0,0 +1,149 @@
+//===- GoogleBenchmarkMain.cpp---------------------------------------------===//
```

Comment:
Recommand to centralize this title and keep the number of characters per line to 80.

Reply:
fixed